README.md

```diff
@@ -7,17 +7,53 @@ Advanced topic group AI: Ket-Hwa, Tingxu, Xiaoting, Zhuoyan
 
 Advanced topic group multiplayer: Jagger, Alli, Sean
 
-# game premise:
+## game premise:
 
 It is a 2D pixel-art platformer where cooperation is necessary. Players control several characters bound by a rope, to conquer the majestic and treacherous slopes of a mysterious mountain, you must master the art of collaborative climbing to reach the top.
```

Comment:
Need some more details about the game. How big is the map? What actions can the player take? What obstacles are there? How many distinct types?, etc.